src/pipelines/epidemiology/config.yaml

```diff
@@ -375,6 +375,11 @@ sources:
     fetch:
       - url: "https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties.csv"
 
+  # Data sources for US_NY level 3 (NYC only)
+  - name: pipelines.epidemiology.us_ny_authority.NYCHealthDataSource
```

Comment:
Unrelated thought: it might be worth having an optional, short description for each data source as well.

Reply:
We add an entry to the README for all data sources, but I forgot to do it here! I'll do it in another PR